mllib/src/main/scala/org/apache/spark/ml/Predictor.scala

```diff
@@ -145,6 +145,10 @@ abstract class PredictionModel[FeaturesType, M <: PredictionModel[FeaturesType,
   /** @group setParam */
   def setPredictionCol(value: String): M = set(predictionCol, value).asInstanceOf[M]
 
+  /** Returns the number of features the model was trained on. Default: -1 */
+  @Since("1.5.1")
```

Comment:
Well, we're adding those incrementally, so you're right to go ahead and add it.  The version should be the next release 1.6.0 (not the little patch releases).